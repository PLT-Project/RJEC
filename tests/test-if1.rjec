func main() {
  if true {
    printi(321);
    printi(888);
  }  
  prints("Hi friends");

}