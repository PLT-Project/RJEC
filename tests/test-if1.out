321
888
Hi friends